app/src/main/res/drawable/ic_baseline_add_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
```

Comment:
These values should be in the dimens.xml file

Reply:
done